src/content/reference/manifest.mdx

```diff
@@ -509,6 +509,41 @@ metadata:
     custom.label/dev: "true"
 ```
 
+#### mode (string, optional)
+
+Defines how `okteto up` will setup the development container and behave within the cluster.
+
+Currently two options are available:
+- `sync`: (default) Sync & run your code in a remote development container.
+- `hybrid`: Run your code on your local computer instead of in a remote development container, while rest of your service components runs in your remote cluster.
+
+Most use-cases should use default `sync` mode, but `hybrid` may be desired to achieve a faster development experience in some scenarios:
+
+|   | `sync` | `hybrid` |
+|---|---|---|
+| Soure code  | Uses Syncthing to sync files bidirectionally with your remote development container | Uses local source code directly, no files syncing needed |
+| `command` context | Runs in the _remote development container_ | Runs on your _local machine_, so traffic need to be rerouted from/to cluster, e.g. using `reverse` option |
+
+##### `hybrid` motivation and example
```

Comment:
@omnomagonz I don't think this kind of docs should "live" here. This makes it harder to navigate the reference. Maybe we need to have a separate page explaining the motivation behind 'okteto up', and then talk in-depth about the modes and when to choose one vs the other?

Reply:
I agree! I'm open to having this in a blog, but then those won't be easy to discover for someone trying to learn more about the feature. Maybe a separate doc page like we did for Catalog. But then that begs the question where do we draw the line on what gets a separate page and what doesn't.